internal/pkg/template/templates/overrides/cdk/README.md

```diff
@@ -0,0 +1,23 @@
+# Welcome to overriding your Copilot generated CloudFormation template with the CDK
+
+This is a CDK project with TypeScript to extend the CloudFormation template that gets 
+deployed with AWS Copilot.
+
+The files of special importance are:
+- `package.json` file holds the version of the CDK Toolkit and Library that Copilot will use to apply the overrides.
+- `stack.ts` file holds the transformations to apply to the CloudFormation template.
+- `bin/override.ts` file holds the entrypoint to the CDK application.
+
+## Troubleshooting
+
+* `copilot [noun] package` preview the transformed template by writing to stdout.
+* `copilot [noun] package --diff` show the difference against the template deployed in your environment
+
+## Under the hood
+The `stack.ts` file uses the [import or migrate an existing AWS CloudFormation template](https://docs.aws.amazon.com/cdk/v2/guide/use_cfn_template.html) 
```

Comment:
???
```suggestion
The `stack.ts` file uses the [`cloudformation-include.CfnInclude` construct](https://docs.aws.amazon.com/cdk/v2/guide/use_cfn_template.html) 
```

Reply:
I updated the wording slightly, take a look to see if it still makes sense